src/cascadia/TerminalSettingsModel/TerminalSettings.h

```diff
@@ -84,21 +85,29 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         INHERITABLE_SETTING(Model::TerminalSettings, til::color, DefaultForeground, DEFAULT_FOREGROUND);
         INHERITABLE_SETTING(Model::TerminalSettings, til::color, DefaultBackground, DEFAULT_BACKGROUND);
         INHERITABLE_SETTING(Model::TerminalSettings, til::color, SelectionBackground, DEFAULT_FOREGROUND);
-        INHERITABLE_SETTING(Model::TerminalSettings, int32_t, HistorySize, DEFAULT_HISTORY_SIZE);
-        INHERITABLE_SETTING(Model::TerminalSettings, int32_t, InitialRows, 30);
-        INHERITABLE_SETTING(Model::TerminalSettings, int32_t, InitialCols, 80);
 
-        INHERITABLE_SETTING(Model::TerminalSettings, bool, SnapOnInput, true);
-        INHERITABLE_SETTING(Model::TerminalSettings, bool, AltGrAliasing, true);
+#define GLOBAL_SETTINGS_INITIALIZE(type, name, ...) \

```

Comment:
Hmm. I wonder if we get more utility by doing the `TerminalSettings` bits with the Control/Core settings macro that I'm doing over in [`dev/migrie/oop/ragnarok`](https://github.com/microsoft/terminal/compare/dev/migrie/oop/ragnarok). That way we could stuff more things into the Global/Profile/Appearance macros as-is. 

Idk that's idle musing without too much thinking about it